### PR TITLE
lxgw-wenkai-screen: init at version 1.330

### PR DESCRIPTION
**Pull request by lebensterben**
## Description of changes

Added [`lxgw-wenkai-screen`](https://github.com/lxgw/LxgwWenKai-Screen), a Traditional Chinese font variation of the existing package
[`lxgw-wenkai`](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=lxgw).

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by lebensterben** [comment]
I probably need to rebase or wait for #309462 to be merged.

**Comment by lebensterben** [comment]
This is ready to be reviewed. Thanks.

**Review by pluiedev — APPROVED** [review]
LGTM except maybe for the use of `rec` instead of `finalAttrs`, but that's as uselessly nitpicky as one can get

**Comment by lebensterben** [comment]
I just made the changes and tested it.

**Review by pluiedev — APPROVED** [review]

**Comment by lebensterben** [comment]
would this PR be accepted?

**Review by pluiedev — APPROVED** [review]
It looks good to me but I don't have commit access so my approval doesn't really mean anything lol

**Comment by lebensterben** [comment]
@NickCao Would you review this PR please?

File: pkgs/by-name/lx/lxgw-wenkai-screen/package.nix
Comment (NickCao) [inline pkgs/by-name/lx/lxgw-wenkai-screen/package.nix:28]:
```suggestion
  meta = {
    description = "LXGW WenKai font optimized for Screen Reading";
    homepage = "https://github.com/lxgw/LxgwWenKai-Screen";
    license = lib.licenses.ofl;
    platforms = lib.platforms.all;
    maintainers = with lib.maintainers; [ lebensterben ];
  };
```

**Review by NickCao — COMMENTED** [review]
Please format the file with nixfmt and squash the commits, then we would be good to go.

**Comment by lebensterben** [comment]
@NickCao Could you do a squash merge on GitHub interface? I have some trouble trying to do it correctly.

**Pull request merged**
